obspy/core/trace.py

```diff
@@ -2779,6 +3241,207 @@ def _data_sanity_checks(value):
                "arrays are allowed for initialization.") % str(value.shape)
         raise ValueError(msg)
 
+class FrequencyDomainTrace(BaseTrace):
+
+    def ifft(self):
+        tr = Trace()
+        tr.stats = Stats()
+        tr.stats.starttime = self.stats.starttime
+        tr.data = fftpack.irfft(self.data)
```

Comment:
I know this is faster than numpy's `rfft()` but it has this extremely awkward return type that is not a complex array as might be expected but a float array with: [real part of zero frequency, real part of f_1, complex part of f_1, real part of f_2, complex part of f_2, ...]. I would suggest to either convert this to a proper complex array - I'm not sure this is possible without a copy - or just use numpy's `rfft` for now. Other opinions?